### DISCUSSION TITLE
pymobiledevice3 seems to hang after  remote tunneld

### DISCUSSION THREAD

**Discussion by Lautarob1** [doronz88/pymobiledevice3#902] [Q&A]
While trying to connect an iOS device to my Windows computer, I used:
python -m pymobiledevice3 remote tunneld and got the following reply:
INFO:     Started server process [8484]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://127.0.0.1:49151 (Press CTRL+C to quit)

Now, the only thing I can do is to quit. So what Im supposed to do so I can connect my iOS device and be able to communicate with it?

**Comment by doronz88** [comment]
This is exactly the expected behvaior. It should just work

**Reply by Lautarob1** [reply]
Thanks!!
Now, once connected as it is, how can I communicate with the device via HTTP? I haven't seen any examples of this or did I overlook them? please help me with this. 
On the other hand, is usbmuxd required / necessary to extract info from iOS devices running iOS >= 17?

**Reply by doronz88** [reply]
usbmux is only a recommended option for ios>=17.4 devices and not mandatory.
you should now be able to simply run any `developer` commands seamlessly

**Reply by Lautarob1** [reply]
Thank You!
